rskj-core/src/main/java/co/rsk/net/discovery/DiscoveryEvent.java

```diff
@@ -38,4 +39,12 @@ public PeerDiscoveryMessage getMessage() {
     public InetSocketAddress getAddress() {
         return address;
     }
+
+    @Override
+    public String toString() {
+        return new ToStringBuilder(this)
+                .append("message", this.message.toString())
+                .append("address", this.address.toString())
```

Comment:
I wouldn't explicitly call toString() here and below

Reply:
Done